hg-fast-export.py

```diff
@@ -122,6 +127,27 @@ def get_author(logmessage,committer,authors):
       return r
   return committer
 
+def generate_git_lfs_pointer(filename):
+  size = 0
+  hasher = hashlib.sha256()
+  with open(filename, "rb") as bfile:
+    buf = bfile.read(1024*1024*1024)
+    while buf:
+      hasher.update(buf)
+      size = size + len(buf)
+      buf = bfile.read(1024*1024*1024)
+  digest = hasher.hexdigest()
+  try:
+    os.makedirs("lfs/objects/%s/%s" % (digest[0:2],digest[2:4]))
+  except:
+    pass
+
```

Comment:
There are 2 reason why I'm reading the full large file in this case. 

One is to generate the file pointer for Git-LFS, I do this by following the [Spec](https://github.com/github/git-lfs/blob/master/docs/spec.md) and we need to provide an oid and it currently only support sha256 so it is fair to assume we need to rehash the file.

Next, we need to copy the file into it's proper folder which is also sha256 based. We do this only to make sure it will be properly uploaded to the new server once the convertion done.